examples/mandelbrot/mandelbrot.pony

```diff
@@ -73,7 +73,7 @@ class val Config
   let outpath: (FilePath | None)
 
   new val create(env: Env) ? =>
-    let cs = CommandSpec.leaf("gups_opt",
+    let cs = CommandSpec.leaf("run",
```

Comment:
The mandelbrot example needed to be fixed and I noticed in the process that the mandelbrot example had been copied from the gups_opt example without changing the command accordingly. No need to include this, but a change I figured as minor enough that it could be cleaned up in the process. Let me know if you want it removed.

Reply:
I think at this point as long as it is noted in the PR that is fine.